tool/data/config/metrics.go

```diff
@@ -29,6 +31,17 @@ func (config *Metrics) ToMap(ctx *runtime.Context) (string, map[string]interface
 		}
 	}
 
+	if config.AggregationDimensions == nil && ctx.WantAggregateDimensions {
```

Comment:
So this assume that the new added option should not coexist with other option like `WantEC2TagDimensions`? Do we ensure that in the wizard?

Reply:
This doesn't prevent `WantEC2TagDimensions` from coexisting. It just adds a separate prompt to add the `AggregationDimensions`. I believe that based on the documentation for setting up the config for metric explorer (https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch-Metrics-Explorer.html), both the `WantEC2TagDimensions` and `WantAggregateDimensions` options are required during setup, but I felt that keeping the two options separate made sense if they weren't configuring with that purpose in mind. The default would include both.